articles/native-platforms/android/03-session-handling.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Auth0 Android Quickstarts - 3. Session Handling
```

Comment:
The title does not display nice. Can you please change it to just `Session Handling`?

This is how it looks:
![image](https://cloud.githubusercontent.com/assets/11715799/16834113/0a1a37a8-49bb-11e6-8e50-95e96bffb2f8.png)

This is how it should look:
![image](https://cloud.githubusercontent.com/assets/11715799/16834122/1479d532-49bb-11e6-9fd5-8803e755e8cb.png)